platform/android/export/export.cpp

```diff
@@ -683,6 +683,22 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		return enabled_plugins;
 	}
 
+    static Error store_in_gradle_project(const String &p_path, const Vector<uint8_t> &p_data, int compression_method = Z_DEFLATED) {
+        //use dir_access.cpp and file_access.cpp to write the file
+        size_t dir_point = p_path.rfind("/");
+        String dir = p_path.substr(0, dir_point); //gets the directory of the file
+        if(!DirAccess::exists(dir)){ //makes an asset directory
```

Comment:
Keep the new code in a consistent style with the old:
`if (!DirAccess::exists(dir)) {`